projects/canopy/src/lib/progress-indicator/progress-indicator.component.html

```diff
@@ -0,0 +1,12 @@
+<lg-progress-journey
+  [max]="max"
+  [value]="value"
+  [showAsPercentage]="showAsPercentage"
+  [stepsPrefix]="stepsPrefix"
```

Comment:
It would  great to add some tests to ensure that these inputs have been set correctly as well. Just thinking again if someone were to come along and remove one of them, or change the value that's being passed in to something hardcoded or programmatically set but not from the input on the component class - it would break the API of the component.